docs/index.rst

```diff
@@ -42,6 +42,20 @@ alternatively::
 
     config.include('pyramid_storage.s3')
 
+To use GCS file storage instead of storing files locally on your server (the default assumption)::
+
+    pyramid.includes =
+        pyramid_storage.gcloud
+
+alternatively::
+
+    config.include('pyramid_storage.gcloud')
+
+We're supporting these authentication methods for GCS::
+
+* JSON credentials file (requires a credentials file to be deployed to the env you're running in, as well as the ``credentials`` argument pointing at the path.).
+* Application Default Credentials `ADC`_ (this makes the ``credentials`` argument unnecessary, but may require the ``project`` arg, e.g. when not running in GKE).
```

Comment:
Should we mention here that it wouldn't make sense to use both the JSON credentials (`gcloud.credential`) _and_ the ADC option (`gcloud.project`)? Unless there _is _ a situation where we'd want to set both.

Reply:
sure, why not, added a comment :slightly_smiling_face: